projects/wave-core-new/src/lib/layers/symbology/symbology.model.ts

```diff
@@ -90,12 +88,12 @@ export abstract class Symbology {
 
 export abstract class VectorSymbology extends Symbology {
     static fromVectorSymbologyDict(dict: VectorSymbologyDict): VectorSymbology {
```

Comment:
I guess there is not such thing as a vector symbology dict anymore? cf. other PR

Reply:
yeah there is no such dict but there is still a VectorSymbology class that encapsulates common behavior which is good